### PR TITLE
Handle click events on relative links.

### LINKED ISSUE #85: Should support opening relative links to markdown
On January 30, 2013 relative links in markup files was added to github, https://github.com/blog/1395-relative-links-in-markup-files

Clicking on `[a relative link](other_file.md)` does not work.

**Env**
markdown-preview 0.71.0 in Atom 0.96.0 on Mac OS X 10.9.2.

**Actual Result**
No action.

**Expected Result**
A new preview tab with the linked md file.

**Additional Details**
Github Pages now supports relative links, as documented December 5, 2016 https://github.com/blog/2290-relative-links-for-github-pages

### PR DESCRIPTION
**Pull request by aroneiermann**
Probably this fix #85.
If click on a relative link open the file and if it is a markdown file open the markdown preview.

### CODE REVIEW

**Comment by 50Wliu** [comment]
Please add specs for this.

**Comment by juaron** [comment]
Would this work for you?

File: spec/markdown-preview-view-spec.coffee [hunk 1]
```diff
@@ -38,6 +38,8 @@ describe "MarkdownPreviewView", ->
       preview.showError("Not a real file")
       expect(preview.text()).toContain "Failed"
 
+    it "Handle click events on relativ links"
```
Comment (davidhesselbom) [inline spec/markdown-preview-view-spec.coffee]: relative

Comment (juaron) [inline spec/markdown-preview-view-spec.coffee]: Thanks.

**Comment by 50Wliu** [comment]
It looks like you have just an empty test that doesn't do anything.  Please see the [specs styleguide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#specs-styleguide) for more information.

**Comment by juaron** [comment]
I am sorry. But somebody other needs to write this test. I will not provide it.

**Comment by kevinsawicki** [comment]
Closing this out, it does not appear to be in a good state right; it is missing specs, adding a new library dependency (string) that does not appear to be needed, and it has conflicts that need to be resolved.

I'm going to close it out for now, please comment once you are able to address these issues.

File: lib/markdown-preview-view.coffee [hunk 2]
```diff
@@ -6,8 +6,10 @@ Grim = require 'grim'
 _ = require 'underscore-plus'
 fs = require 'fs-plus'
 {File} = require 'pathwatcher'
+String = require 'string'
```
Comment (kevinsawicki) [inline lib/markdown-preview-view.coffee]: I'm not sure this is needed, strings are built-in and shouldn't require a third-party library.

**Comment by pfhm** [comment]
Ok, I have zero coffescript experience, but because I need this badly and this appears to be achievable with very little code...

Let's see if I can contribute.
Can I add speccs? Where? Should I open a new issue? should I fork the project, pull these commits then submit a new pull request with a more detailed description of the feature?
Or are there clear guidelines for submitting feature specs? (Link please)

**Comment by pfhm** [comment]
Oh sorry. Thought the above link was relevant to tests only...  Reading the guidelines.

**Pull request closed without merging**